servers/rendering/renderer_scene_occlusion_cull.h

```diff
@@ -83,6 +83,10 @@ class RendererSceneOcclusionCull {
 				Vector3 corner = Vector3(p_bounds[0] * c.x + p_bounds[3] * nc.x, p_bounds[1] * c.y + p_bounds[4] * nc.y, p_bounds[2] * c.z + p_bounds[5] * nc.z);
 				Vector3 view = p_cam_inv_transform.xform(corner);
 
+				if (p_cam_projection.is_orthogonal()) {
+					min_depth = MIN(min_depth, view.z);
+				}
```

Comment:
Will this orthogonal case be susceptible to the same problem as the other PR? Namely that a large AABB may come close to the camera z near far off from the view frustum?

I'm not absolutely sure this will be the best way to handle this in the orthogonal case, as is.

Reply:
This ends up being the exact same math as in the other PR for the the orthogonal case. So the same issue will exist yes, but I don't think think the we have any other option at the moment.

Reply:
You can probably do something like identify the closest vert on the AABB, then intersect the 3 line segments with the frustum edges to find the real closest point, if the AABB straddles the view frustum. This might be simpler as the view is orthogonal.

Whether this will lead to that many more being rejected in practice I don't know, as there are the other snags with the algorithm with slanted AABBs.

But something that could be left to follow up.

Reply:
That makes sense. Will look into it.